src/Uno.UI.Composition/Composition/CompositionSpriteShape.skia.cs

```diff
@@ -126,6 +126,12 @@ private static SKPaint TryCreateAndClearPaint(in Visual.PaintingSession session,
 					paint.PathEffect.Dispose();
 					paint.PathEffect = null;
 				}
+
+				if (paint.ImageFilter != null)
```

Comment:
Is this something that can be tested? A simple screenshot validation can do the trick.